wp-content/themes/currentorg/partials/projects-single-holder.php

```diff
@@ -1,6 +1,8 @@
 <div class="projects-single-layout">
-	<button class="projects-single-close">Close</button>
-	<div class="projects-single-holder">
-		<?php include( locate_template( 'partials/projects-single-held.php', false, false ) ); ?>
+	<div class="projects-single-interior-container">
+		<button class="projects-single-close btn">Close</button>
+		<div class="projects-single-holder">
+			<?php include( locate_template( 'partials/projects-single-held.php', false, false ) ); ?>
```

Comment:
On initial page load, with no items clicked and no URL params set, we get a post loaded here. Should one be?

Reply:
@benlk It's been like this since we got started on the base of the projects list view and single project view templates. I think showing _a_ post is better than not showing one?

Reply:
And it matches the display at https://jobs.current.org/jobseeker/search/results/, so I guess there isn't any problem. It just seems weird to me, that's all.